common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java

```diff
@@ -133,11 +133,7 @@ public abstract boolean removeAccount(final OperationParameters parameters)
                         .setMultipleCloudAware(requestAuthority.mMultipleCloudsSupported)
                         .setSlice(requestAuthority.mSlice);
             }
-
-            if(builder instanceof MicrosoftStsAuthorizationRequest.Builder){
-                ((MicrosoftStsAuthorizationRequest.Builder)builder).setTokenScope(TextUtils.join(" ", parameters.getScopes()));
-            }
-
+            
```

Comment:
Not quite this easy, I'm afraid 😞 

Resolving this issue by removing this setter call reintroduces the bug reported in #524. I suggest taking a look at #527 to see why this was introduced.

In essence, what we need to do is not include this value _in the authorization URL_ but still have it available on this object. I suspect this means marking `mTokenScope` as `transient` so that it is not serialized into the request, rather than removing the setter call to it.

Reply:
Pushed the fix. Thanks

Reply:
Thanks @shahzaibj - Can I get you to run the Android Studio code formatter and remove the stray whitespace in this file?

(It's in the top-menu under `Code` > `Reformat Code`)

I don't see any import issues, but you could run `Optimize Imports` as well, good habit for code cleanliness in general

Reply:
Pushed